### PR TITLE
Upgrade agent plugin for ElasticJob 3.0.0 GA version

### PR DESCRIPTION
**Pull request by TeslaCN**
The current plugin is for ElasticJob 3.0.0-alpha. I am going to upgrade it to ElasticJob 3.0.0.

### Upgrade an agent plugin to support ElasticJob 3.0.0
- [x] Upgrade test case for the plugin.
- [x] Add a component id in [the component-libraries.yml](https://github.com/apache/skywalking/blob/master/oap-server/server-bootstrap/src/main/resources/component-libraries.yml)
- [x] Add a logo in [the UI repo](https://github.com/apache/skywalking-rocketbot-ui/tree/master/src/views/components/topology/assets)

- [x] Update the [`CHANGES` log](https://github.com/apache/skywalking/blob/master/CHANGES.md).

### CODE REVIEW

File: docs/en/setup/service-agent/java-agent/Supported-list.md [hunk 1]
```diff
@@ -98,7 +98,7 @@ metrics based on the tracing data.
 * [Sentinel: The Sentinel of Your Microservices](https://github.com/alibaba/Sentinel) 1.7.0 -> 1.8.1
 * Scheduler
   * [Elastic Job](https://github.com/elasticjob/elastic-job) 2.x
-  * [Apache ShardingSphere-Elasticjob](https://github.com/apache/shardingsphere-elasticjob) 3.0.0-alpha
+  * [Apache ShardingSphere-Elasticjob](https://github.com/apache/shardingsphere-elasticjob) 3.x
```
Comment (wu-sheng) [inline docs/en/setup/service-agent/java-agent/Supported-list.md:101]: Does ShardingSphere confirm this break change? This is pretty new plugin, which was added months ago.

Comment (TeslaCN) [reply docs/en/setup/service-agent/java-agent/Supported-list.md:101]: This plugin was for 3.0.0-alpha. The ElasticJob 3.0.0 GA released so I think the alpha can be replaced.

Comment (TeslaCN) [reply docs/en/setup/service-agent/java-agent/Supported-list.md:101]: Or maybe I should rename it to 3.0.0 rather than 3.x.

Comment (wu-sheng) [reply docs/en/setup/service-agent/java-agent/Supported-list.md:101]: SkyWalking doesn't have preference or requirement. We just ask you to be clear for end users. Nothing more.

This comment is for confirming only for thr future.

Comment (TeslaCN) [reply docs/en/setup/service-agent/java-agent/Supported-list.md:101]: Understood.

File: apm-sniffer/apm-sdk-plugin/elasticjob-3.x-plugin/pom.xml [hunk 2]
```diff
@@ -42,5 +42,11 @@
```
[the remark this answers is not in the document]
Comment (TeslaCN) [reply apm-sniffer/apm-sdk-plugin/elasticjob-3.x-plugin/pom.xml]: Fix it in ElasticJob later.

File: apm-sniffer/apm-sdk-plugin/elasticjob-3.x-plugin/pom.xml [hunk 3]
```diff
@@ -42,5 +42,11 @@
             <version>${elasticjob.version}</version>
             <scope>provided</scope>
         </dependency>
+        <dependency>
```
[the remark this answers is not in the document]
Comment (TeslaCN) [reply apm-sniffer/apm-sdk-plugin/elasticjob-3.x-plugin/pom.xml:45]: Fix it in ElasticJob later.

**Review by wu-sheng — APPROVED** [review]
Good to me.

File: apm-sniffer/apm-sdk-plugin/elasticjob-3.x-plugin/pom.xml (hunk 3 above)
[the remark this answers is not in the document]
Comment (wu-sheng) [reply apm-sniffer/apm-sdk-plugin/elasticjob-3.x-plugin/pom.xml:45]: Provided scope means not SkyWalking concern.

**Pull request merged**